definitions/npm/chai_v4.x.x/flow_v0.25.0-/chai_v4.x.x.js

```diff
@@ -158,7 +159,11 @@ declare module "chai" {
     matchSnapshot: (lang?: any, update?: boolean, msg?: any) => ExpectChain<T>
   };
 
-  declare function expect<T>(actual: T, message?: string): ExpectChain<T>;
+
+  declare var expect: {
+    <T>(actual: T, message?: string): ExpectChain<T>,
```

Comment:
This will probably need be scoped to an higher version of Flow, support for callable generics was added later if I'm not mistaken.

Reply:
I've ran
`node cli/dist/cli.js run-tests --onlyChanged --numberOfFlowVersions 100`

And had the following messages:
`flow-typed only supports flow 0.53.0 and newer`

Tests are passing for 0.53.0, if it's ok for you this module can be moved to 0.53.0- ?